core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -243,7 +243,8 @@
             <source type="image/png"
                     [srcset]="this.getStaticImageUrl('/sidebar/learn-math-logo.png')">
             <img [src]="getStaticImageUrl('/sidebar/learn-math-logo.png')"
-                 class="oppia-submenu-image"
+                 height="80px"
+                 width="64.54px"
```

Comment:
@Patel-Muhammad The PR is still facing the same error, do we need `px` suffix here? If possible we can use `svg` image?